docs/faq.md

```diff
@@ -2,41 +2,51 @@
 
 Save the notebook, and you get two copies of the notebook: the original `*.ipynb` file, together with its paired text representation.
 
+Read more about how to use Jupytext in the [documentation](using-server.md).
+
 ## Which Jupytext format do you recommend?
 
-I tend to use the Markdown format for notebooks that contain more text than code, as Markdown documents are conveniently edited in IDEs and also well rendered on GitHub.
+Notebooks that contain more text than code are best represented as Markdown documents. These are conveniently edited in IDEs and are also well rendered on GitHub.
 
-Saving notebooks as scripts is a convenient choice when you want to refactor your notebook in an IDE (or import it in another notebook, etc). Use the `percent` format if you prefer to get explicit cell markers (compatible with VScode, PyCharm, Spyder, Hydrogen...). If you prefer to get the minimal amount of cell markers, go for the `light` format.
+Saving notebooks as scripts is an appropriate choice when you want to act on the code (refactor the code, import it in another script or notebook, etc). Use the `percent` format if you prefer to get explicit cell markers (compatible with VScode, PyCharm, Spyder, Hydrogen...). And if you prefer to get the minimal amount of cell markers, go for the `light` format.
 
 ## Can I see a sample of each format?
 
 Go to [our demo folder](https://github.com/mwouts/jupytext/tree/master/demo) and see how our sample `World population` notebook is represented in each format.
 
 ## Can I edit the paired text file?
 
-Yes! And when you're done, refresh the notebook in Jupyter. Refreshing will bring the latest changes to your notebook.
+Yes! When you're done, reload the notebook in Jupyter. There, you will see the updated input cells combined with the matching output cells from the `.ipynb` file.
+
+## Do I need to close my notebook in Jupyter?
+
+No, you don't (*). You can edit the paired text file and simply refresh your navigator to reload the updated input cells. When you refresh the notebook, the kernel variables are preserved, so you can continue your work where you left it.
+
+(*) Please read about Jupyter's autosave below.
 
 ## How do paired notebooks work?
 
-The `.ipynb` file contains the full notebook. The paired text file only contains the input cells and selected metadata. When the notebook is loaded by Jupyter, input cells are loaded from the text file, while the output cells and the filtered metadata are restored using the `.ipynb` file.
+The `.ipynb` file contains the full notebook. The paired text file only contains the input cells and selected metadata. When the notebook is loaded by Jupyter, input cells are loaded from the text file, while the output cells and the filtered metadata are restored using the `.ipynb` file. When the notebook is saved in Jupyter, the two files are updated to match the current content of the notebook.
```

Comment:
You can click on either file to open it into the notebook editor, edit it and run it there, and when you save it, both files will be updated using the appropriate file format.

?What are likely problems if you have both files open at the same time?

Reply:
Exactly the same as if you are editing the same document in two editors. In short, as long as you modify just one of the two documents you're safe. You may find the autosave a bit annoying, but it won't hurt as Jupytext implements timestamps checks. Read more on this in the next Q/A.